lib/idx/remediators/Base/Remediator.ts

```diff
@@ -130,7 +130,7 @@ export class Remediator {
   }
 
   // Get inputs for the next step
-  private getInputs() {
+  private getInputs(): Input[] {
```

Comment:
If we also mark all of the `getInput*()` methods with `Input` as the return type, it would help us ensure the `Input` type is fully up to date with possible fields.